.circleci/config.yml

```diff
@@ -24,8 +24,20 @@ jobs:
           command: ./gradlew --stacktrace lint checkstyle
       - android/save-gradle-cache
       - android/save-lint-results
+  Test:
+    executor:
+      name: android/default
+      api-version: "27"
+    steps:
+      - checkout
+      - copy-gradle-properties
+      - android/restore-gradle-cache
+      - run:
+          name: Test
+          command: ./gradlew --stacktrace test
```

Comment:
`test` runs the tests for both build types, it seemed more concise than using `./gradlew testDebug testRelease`.